### ISSUE TITLE
Enhancement: one touch PAUSE/RESUME and one touch SCREENSHOT

### ISSUE THREAD

**Issue by owenduffy** [DiSlord/NanoVNA-D#84]
It would be very convenient if one could PAUSE / RESUME by touching an icon on the screen.

It would be very convenient if one could take a SCREENSHOT forcing AUTONAME for the duration and automatic RESUME by touching an icon on the screen.

![nanovnah4_20240619_144711](https://github.com/DiSlord/NanoVNA-D/assets/4297310/9b964380-6087-450d-9deb-0d19e7f12771)

The above screenshot has had a P and S added at lower right as a suggested interface to these two functions. You could change the colour of the P to show paused or not, and change the colour of the S to show 'working'.

Owen

**Comment by DiSlord** [comment]
If click on BW text at bottom, must run screenshot procedure
If auto name enabled - auto or ask name

**Comment by owenduffy** [comment]
I thought I checked by touching the left margin of the screen that it was unused, but I see it is now used for adjusting scale... though in a little exploration I kept getting into a state where I needed to use the menu to straighten things out.

I did not know of the BW trick.

When I take a screenshot, I usually PAUSE so that I am sure the screenshot is going to have what I want to show, then take the screenshot and if necessary RESUME scan. It is a lot of keystrokes, and for a common operation I thought was I proposed would be very convenient.

Owen
